src/components/NcDialog/NcDialog.vue

```diff
@@ -103,7 +103,8 @@ export default {
 				<nav v-if="hasNavigation"
 					class="dialog__navigation"
 					:class="navigationClasses"
-					:aria-labelledby="navigationId">
+					:aria-label="navigationAriaLabelAttr"
+					:aria-labelledby="navigationAriaLabelledbyAttr">
```

Comment:
```suggestion
					:aria-labelledby="navigationAriaLabelledbyAttr || navigationId">
```

For so it is not breaking?

Reply:
It is done by `navigationAriaLabelledbyAttr`